symfony/plugins/orangehrmOpenidAuthenticationPlugin/modules/openidauth/actions/openIdCredentialsAction.class.php

```diff
@@ -141,14 +152,16 @@ public function showFlashMessage($flag) {
         }
     }
 
-    public function getAuthProviderObj($providerType, $provider, $authProviderDetails) {
+    public function getAuthProviderObj($providerType, $provider, $authProviderDetails)
+    {
         switch ($providerType) {
             case AuthProviderExtraDetails::OPEN_ID:
                 $openIdAuthProvider = new OpenIdAuthProvider();
                 $flag = $openIdAuthProvider->validateUser($provider);
                 break;
             case AuthProviderExtraDetails::GOOGLE_PLUS:
                 $googleAuthProvider = new GoogleAuthProvider();
+                $googleAuthProvider->setOption($_GET);
```

Comment:
can't we get this as param as $request->getParams() ?

Reply:
baseOpenIdAction  doesn't provide $request->getParamameters() but it provide  $this->getRequest()->getParameter('code'); we can use this.